src/component/feature/FeatureToggleList/FeatureToggleListItem/FeatureToggleListItem.tsx

```diff
@@ -0,0 +1,171 @@
+import { memo } from 'react';
+import classnames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Chip, ListItem, Tooltip } from '@material-ui/core';
+import { Undo } from '@material-ui/icons';
+import TimeAgo from 'react-timeago';
+import { IAccessContext } from 'contexts/AccessContext';
+import StatusChip from 'component/common/StatusChip/StatusChip';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { UPDATE_FEATURE } from 'component/providers/AccessProvider/permissions';
+import { IFeatureToggle } from 'interfaces/featureToggle';
+import { IFlags } from 'interfaces/uiConfig';
+import { getTogglePath } from 'utils/routePathHelpers';
+import FeatureStatus from 'component/feature/FeatureView/FeatureStatus/FeatureStatus';
+import FeatureType from 'component/feature/FeatureView/FeatureType/FeatureType';
+import useProjects from 'hooks/api/getters/useProjects/useProjects';
+import PermissionIconButton from 'component/common/PermissionIconButton/PermissionIconButton';
+import { styles as commonStyles } from 'component/common'; // FIXME: remove
+import { useStyles } from './styles'; // FIXME: cleanup
+
+interface IFeatureToggleListItemProps {
+    feature: IFeatureToggle;
+    onRevive?: (id: string) => void;
+    hasAccess: IAccessContext['hasAccess'];
+    flags?: IFlags;
+    className?: string;
+}
+
+export const FeatureToggleListItem = memo<IFeatureToggleListItemProps>(
```

Comment:
For an indent less, and to communicate the memoization, we could use memo on its own:

```
const FeatureToggleListItem: VFC<IFeatureToggleListItemProps> = ...

export const FeatureToggleListItemMemo = React.memo(FeatureToggleListItem);
```

Reply:
How large is the performance gain here?

Reply:
I'll leave it for now, because we will deprecate this component soon.